doc/news/_preparation_next_release.md

```diff
@@ -82,6 +82,11 @@ The following section lists news about the [modules](https://www.libelektra.org/
 
 - We fixed an [use after free bug in the plugin](https://issues.libelektra.org/3561). _(René Schwaiger)_
 
+### GOpts
+
+- The `gopts` plugin can now be configured to use custom `argv` and `envp` values. _(Klemens Böswirth)_
+  WARNING: This feature is experimental and may be removed, replaced or otherwise changed in a future release.
```

Comment:
This is not necessary to say. Within `0.9.*` we can change anything anyway but not afterwards (depending on the status of the plugin).

What would be relevant to say if we actually plan to remove it and I hope we do as I do not like a global symbol for a setup. Ideally it should be in kdbOpen.